steps/identity/appellant-international-contact-details/content.cy.json

```diff
@@ -1,12 +1,21 @@
 {
   "titleHead": "Manylion cyswllt",
-  "title": "Rhowch eich manylion cyswllt",
-  "subtitle": "Fel y penodai, rhowch eich manylion cyswllt chi.",
+  "title": {
+    "withoutAppointee": "Nodwch eich manylion cyswllt",
+    "withAppointee": "Nodwch eich cyfeiriad"
+  },
+  "subtitle": {
+    "withoutAppointee": "As the appellant, enter your contact details.",
```

Comment:
is the translation for this not given above in what was line 4?

Reply:
Nope, the previous content used here was actually appointee related, I grabbed this from the regular old appellantContactDetails content as it should be the same.